README.md

```diff
@@ -1,70 +1,113 @@
-# Getting Started with Create React App
+<a id="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
 
-## Available Scripts
+  <img src="https://user-images.githubusercontent.com/108806646/203941212-a33b97da-6d84-4649-b763-4e8191c24805.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-In the project directory, you can run:
+  <h3><b>README</b></h3>
 
-### `npm start`
+</div>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+# 📗 Table of Contents
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- [📗 Table of Contents](#-table-of-contents)
+- [Air-Pollution](#air-pollution)
+  - [Built With](#built-with)
+  - [Live Demo](#live-demo)
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Available Scripts](#available-scripts)
+  - [Author](#author)
+  - [Contributing](#contributing)
+  - [Acknowledgments](#acknowledgments)
+  - [License](#license)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+# react-redux-capstone-proj
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+**Air-Pollution** This is a SPA mobile web application to check a list of metrics for the air quality of a specified area.and states
+
+## Built With
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<ul>
+  <li><a href="https://reactjs.org/">React.js</a></li>
+  <li><a href="https://www.ecma-international.org/">JavaScript</a></li>
+</ul>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+## Live Demo
 
-### `npm run eject`
+- [Live Demo Link](https://resonant-raindrop-6d1bf0.netlify.app)
+- [Video Link](https://www.loom.com/share/391a7a0ae1ab43918a9fb69f1fc38c25)
+- [Video Link 2] (https://resonant-raindrop-6d1bf0.netlify.app)
```

Comment:
While you did a good job recording the video it is not audible I think your mic has a problem work on it and record another video. It would be nice if you highlight a piece of code and also talk about  feature you added that you are most proud of. I suggest you kindly record your video based on the following requirements:

- The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.